mdpow/analysis/ensemble.py

```diff
@@ -516,16 +516,17 @@ def run(self, start=None, stop=None, step=None):
         self._prepare_ensemble()
         for self._key in ProgressBar(self._ensemble.keys(), verbose=True):
             self._setup_system(self._key, start=start, stop=stop, step=step)
-            self._prepare_universe()
-            self._single_universe()
-            for i, ts in enumerate(ProgressBar(self._trajectory[self.start:self.stop:self.step], verbose=True,
+            try:
+                self._single_universe()
+            except NotImplementedError:
+                for i, ts in enumerate(ProgressBar(self._trajectory[self.start:self.stop:self.step], verbose=True,
                                                postfix=f'running system {self._key}')):
                 self._frame_index = i
                 self._ts = ts
                 self.frames[i] = ts.frame
                 self.times[i] = ts.time
                 self._single_frame()
-            self._conclude_universe()
+                raise
```

Comment:
why raise??

where is conclude?

Reply:
Based on your discussion with Alia it was decided that conclude universe was no longer necessary for the functionality before or after splitting the back end run methods. I will try to find the conversation and post a screenshot here shortly.

Reply:
https://github.com/Becksteinlab/MDPOW/pull/199#issuecomment-1241307425

Reply:
Ok, that’s goid then.

Reply:
But why the raise?

Reply:
Sorry, I removed raise again and fixed the expected indent error, so it should be okay now. I could not figure out how to get it to work properly after making the changes, but I think the error is in how my development environment is configured, or how _single_universe() is defined in the EnsembleAnalysisClass I was using to test. The original changes were made using an older version of Python, so I am going to start my development environment over from scratch, redo the usage testing with a simple analysis class, and see if any other changes are required.

Reply:
Your version of Python or IDE should not affect how Python code is interpreted. As long you use Python ≥ 3.7 everything should be fine.